BraveShared/BraveStrings.swift

```diff
@@ -3822,3 +3822,23 @@ extension Strings {
   public static let invalidDetailsMessage = NSLocalizedString("invalidDetailsMessage", tableName: "BraveShared", bundle: .strings, value: "Invalid details in payment request", comment: "Error message if details don't have the right type or values")
   public static let clientErrorMessage = NSLocalizedString("clientErrorMessage", tableName: "BraveShared", bundle: .strings, value: "Client error", comment: "Client is in an invalid state which caused the error")
 }
+
+extension Strings {
+  public static let urlRedirectsSettings =
+  NSLocalizedString("urlRedirectsSettings", tableName: "BraveShared",
+                    bundle: .strings,
+                    value: "Website Redirects",
+                    comment: "Setting title for option to automatically redirect a url to another url")
+  
+  public static let redditRedirectFooter =
+  NSLocalizedString("redditRedirectFooter", tableName: "BraveShared",
+                    bundle: .strings,
+                    value: "Automatically redirects Reddit links to the older interface (at old.reddit.com). To force the new Reddit interface, use new.reddit.com instead.",
```

Comment:
Do we display this text to users?

If so, it seems odd to talk about `new.reddit.com` since we don't offer that capability, we just offer the ability to redirect to `old.reddit.com`.

Reply:
We do offer that, if you go to `new.reddit.com` it will not redirect,
copied this rule from this extension
https://github.com/tom-james-watson/old-reddit-redirect#whitelisted-domains

Reply:
Sorry, I meant to remove this comment after I saw the list of exclusions later down the PR.

Reply:
Sorry, I meant to remove this comment after I saw the list of exclusions later down the PR.